app/routes/your-claims/view-claim.js

```diff
@@ -22,6 +23,15 @@ module.exports = function (router) {
     getViewClaim(req.params.claimId, decryptedReference, req.params.dob)
       .then(function (claimDetails) {
         var referenceId = referenceIdHelper.getReferenceId(decryptedReference, claimDetails.claim.EligibilityId)
+        var isRequestInfoPayment = claimDetails.claim.Status === 'REQUEST-INFO-PAYMENT'
+        console.log(claimDetails)
+        var addInformation = addInformationHelper(claimDetails.claim.Status,
```

Comment:
Can the name be more descriptive? It's a single function module (not a helper) which returns an array of message objects, getRequiredInformationWarnings?